test/testcafe/spec/BYOL_spec.js

```diff
@@ -46,62 +45,44 @@ test.requestHooks(mock)('unsupported language, set with "language" option, is no
 
   // Check title (login_foo.json)
   await t.expect(pageObject.getFormTitle()).eql('Set up phone authentication');
-
   // Check country dropdown (country_foo.json)
-  const selectOption = await pageObject.form.findFormFieldInput('country')
-    .find('.chzn-container > .chzn-single > span');
-
-  await t.expect(pageObject.getTextContent(selectOption))
-    .eql('United States');
-
+  await t.expect(await pageObject.countryDropdownHasSelectedText('United States')).eql(true);
 });
 
 test.requestHooks(mock)('unsupported language, set with "language" option, can be loaded when assets.baseUrl is set to a path containing the language assets', async t => {
   const pageObject = await setup(t, {
     language: 'foo',
     assets: {
-      baseUrl: '/mocks'
+      baseUrl: 'http://localhost:3000/mocks'
```

Comment:
I found the issue was that the loc function could not load the custom language bundle when just using the absolute path (assuming because v3 is nested within src).